jdaviz/configs/specviz/plugins/unit_conversion/tests/test_unit_conversion.py

```diff
@@ -1,92 +1,278 @@
 import astropy.modeling.models as models
 import astropy.units as u
 import numpy as np
-from specutils.spectra import Spectrum1D
-
+from specutils import Spectrum1D, SpectrumCollection
+from jdaviz.configs.specviz.helper import SpecViz
 from .. import unit_conversion as uc
+import pytest
+from astropy.nddata import VarianceUncertainty, StdDevUncertainty,\
+    InverseVariance, UnknownUncertainty
```

Comment:
Slash has been discouraged for a while. Use parenthesis.
```suggestion
from astropy.nddata import (VarianceUncertainty, StdDevUncertainty,
                            InverseVariance, UnknownUncertainty)
```

Reply:
Did not know that, thanks!